src/utils/config.ts

```diff
@@ -25,6 +25,12 @@ function validatePortGap (portGap: number) {
   }
 }
 
+function validateZoom (zoom: number) {
+  if (typeof zoom !== 'number' || zoom < 0) {
```

Comment:
```suggestion
  if (!Number.isFinite(zoom) || zoom < 0) {
```

如果 zoom 是 NaN 的话，检测不出来

Reply:
原库是允许 NaN 的，相当于没有限制 zoom